presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -282,7 +292,7 @@ public void testBinaryOperatorsBoolean()
     public void testBinaryOperatorsLongLong()
             throws Exception
     {
-        for (Long left : longLefts) {
+        for (Long left : smallLongs) {
```

Comment:
Because of the overflow since `extremeLongs` multiplied by big numbers from `longRights` may cause overflow. In some cases even adding `1` to e.g. `Long.MAX_VALUE` may cause an overflow.
I wanted to avoid adding more logic to the test case (to figure out if this is a case of overflow) and I didn't want to change numbers because these are used in other use cases too.